get_statistics.py

```diff
@@ -97,6 +97,68 @@ def closed_issues(days_ago: int, till: str, repository: str):
     click.echo("Ops: {}".format(aggregated_data["ops"]))
     click.echo("Dev: {}".format(aggregated_data["dev"]))
 
+@click.command()
+@click.option("--days-ago", default=30, help="How many days ago to look for created issues.")
+@click.option("--till", default=None, help="Show results till this date. Expects date in DD.MM.YYYY format (31.12.2021).")
+@click.argument('repository')
+def created_issues(days_ago: int, till: str, repository: str):
+    """
+    Get created issues from the repository and print their count.
+
+    Params:
+      days_ago: How many days ago to look for the issues
+      till: Limit results to the day set by this argument. Default None will be replaced by `arrow.utcnow()`.
+      repository: Repository namespace to check
+    """
+    if till:
+        till = arrow.get(till, "DD.MM.YYYY")
+    else:
+        till = arrow.utcnow()
+    since_arg = till.shift(days=-days_ago)
+
+    next_page = PAGURE_URL + "api/0/" + repository + "/issues?status=all&since=" + str(since_arg.int_timestamp)
```

Comment:
I'm not sure if this will work as you think, `since` is looking for updated issues from the specified time, not for the new ones. So if you only want issues that were created since the date specified you need to further filter them by `date_created` field. See https://pagure.io/api/0/#issues-tab `List project's issues` for more info.